src/main/java/com/blackwatch/diskonect/model/Post.java

```diff
@@ -11,4 +12,24 @@ public record Post(
     @NonNull String content,
     @NonNull Date createdAt,
     @NonNull String slug,
-    @NonNull List<String> assets) {}
+    @NonNull List<String> assets) {
+    public Post(@NonNull Long id, @NonNull String title, @NonNull String content, @NonNull Date createdAt, @NonNull String slug, @NonNull List<String> assets) {
+        this.id = id;
```

Comment:
Just a pointer for you to learn. Java `record` was added only in java 14 and widely used since java 17 (very recent). So not a lot of people know about it. For most people, java means java 8.